### ISSUE TITLE
Been waiting for new version of Effective for months.

### ISSUE THREAD

**Issue by NolanIsDev** [Ladysnake/Effective#179]
I need the 1.19.3 to give my trailer mod pack to be the best out there, I was just wondering when I can expect a 1.19.3 update. Thanks for your time.

**Comment by SpiritFancy** [comment]
SAME! I need it for **1.19.4**

**Comment by GitsurferQwerty** [comment]
Honestly, I understand that the devs are working hard on updating to 1.19.3/1.19.4 but honestly it has been so long since this has been updated that I totally forgot this mod exists. At this rate we're gonna get the Effective update at the end of the year.

**Comment by SpiritFancy** [comment]
> Honestly, I understand that the devs are working hard on updating to 1.19.3/1.19.4 but honestly it has been so long since this has been updated that I totally forgot this mod exists. At this rate we're gonna get the Effective update by 2024.

BY 2024?!?!?!?!??! NOOOOOOOOOOOOOOOOOOOO

**Comment by peepeepoopoocaca** [comment]
> Now 1.20 is out, and Effective is now 3 versions behind. I might as well let this mod go.

Yeah, no. This mod is basically dead. I just asked in the server if the 1.20 version was being worked on and I was told that the creator was "busy working on videos," despite not uploading in like 2 months so, I guess we're screwed.

**Comment by Pyrofab** [comment]
Y'all have no idea how much effort Rat spends on his daytime job, please stop being so dismissive when it comes to a mod he updates on the time he has left after that. It is okay to be pained because a mod you like is in a state of stasis, but it is not acceptable to dogpile on the creator in hopes that they will cave at the cost of their mental health.

I will be closing this issue as it was pretty much destined to be a repository for complaints.
